orttraining/orttraining/core/graph/zero_optimizer_graph_builder.cc

```diff
@@ -214,8 +214,7 @@ void PartitionOptimizerState(
                                                     info);
 
       } else {
-        ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "Unsupported type: ", element_type,
-                        "for initial optimizer moments.");
+        ORT_THROW("Unsupported type: ", element_type, "for initial optimizer moments.");
```

Comment:
nit: space at beginning of "for initial optimizer moments."

Reply:
will fix if there are additional comments to avoid CI delay